src/content/docs/baas/sanity/sanityio-case-study-bettytro.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Sanity.io - Headless CMS
+keywords: development platforms, sanity, headlesscms
+tags: Development Platforms, Sanity, Headless CMS
+---
+
+## Introduction
+
+This assessment will cover my experiences with sanity.io, I will also go in to headless CMS a bit. There is no understanding Sanity without understanding the basics of headless CMS. I will also reflect on why I think you should pick a headless CMS, and why it probably should be Sanity.
```

Comment:
Technical writing does not use I/me as the idea is not to give your opinion but factual details on the subject.

```suggestion
In this case study we will assess sanity.io, as well as discussing the concept of headless CMS...
```